docker/docker-compose.yml

```diff
@@ -1,21 +1,29 @@
 version: "3.3"
 services:
-    geth:
-        image: ethereum/client-go:v1.12.0
-        environment:
-            - DEV_PERIOD
-        ports:
-            - "8545:8545"
-        volumes:
-            - ./gethData/geth_data:/geth_data
-        entrypoint:
-            - geth
-            - --rpc.allow-unprotected-txs
-            - --http
-            - --http.addr
-            - "0.0.0.0"
-            - --dev
-            - --dev.period
-            - $DEV_PERIOD
-            - --datadir
-            - /geth_data
+  # Sets up the genesis configuration for the go-ethereum client from a JSON file.
```

Comment:
can we add a comment to document that this is needed b/c we don't know why `terminalTotalDifficultyPassed` were not correctly set in the default config